src/script/lua_api/l_http.cpp

```diff
@@ -180,11 +198,35 @@ int ModApiHttp::l_request_http_api(lua_State *L)
 
 	return 1;
 }
+
+int ModApiHttp::l_get_http_api(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+
+	lua_newtable(L);
+	HTTP_API(fetch_async);
+	HTTP_API(fetch_async_get);
+	HTTP_API(fetch_sync);
+
+	return 1;
+}
+
 #endif
 
 void ModApiHttp::Initialize(lua_State *L, int top)
 {
 #if USE_CURL
-	API_FCT(request_http_api);
+
+	bool isMainmenu = false;
+#ifndef SERVER
+	isMainmenu = ModApiBase::getGuiEngine(L);
```

Comment:
surprised this doesn't raise a warning, I'd prefer an explicit `!!` or ` != nullptr`.